tests/spirv/pointer-array-2.slang

```diff
@@ -0,0 +1,27 @@
+struct input_t {
+    uint32_t i : 8;
+    uint32_t j : 24;
+};
+
+struct output_t {
+    uint32_t i;
+    uint32_t j;
+};
+
+uniform void *parameters[1000];
+
+//TEST:SIMPLE(filecheck=CHECK): -target spirv
+
+//CHECK: OpEntryPoint
```

Comment:
what else do we want to check in this test?

Reply:
Not much beyond we generate valid spirv.